src/ngx_http_lua_balancer.c

```diff
@@ -297,6 +298,7 @@ ngx_http_lua_balancer_get_peer(ngx_peer_connection_t *pc, void *data)
     bp->socklen = 0;
     bp->more_tries = 0;
     bp->total_tries++;
+    bp->tries = r->upstream->peer.tries;
```

Comment:
Seems we don't need to save `tries` in `bp`.
We can get `r->upstream->peer.tries` in `set_more_tries` also.